ts/routes/graphs/RangeBox.svelte

```diff
@@ -112,6 +112,15 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 <div class="range-box-pad"></div>
 
 <style lang="scss">
+    label {
+        display: inline-flex;
+        align-items: center;
+    }
+
+    input[type="radio"] {
+        margin-right: 0.3em;
```

Comment:
Please use `margin-inline-end` for RTL compatibility.
![image](https://github.com/user-attachments/assets/90fc29e8-5203-482d-971d-352acbea5a3c)